### ISSUE TITLE
a11y: ARIA Label missing

### ISSUE THREAD

**Issue by svalyi-op** [openPetition/opnVote#177]
Internal [Bug Hunt](https://docs.google.com/document/d/1dartgwZFjJgeVdiVeXQbE_muo7mK43dQE4USpqVQzII/edit?tab=t.0)

Calendar Download Icon should get Aria Label (with translations)
